lumigator/backend/backend/services/jobs.py

```diff
@@ -498,7 +432,7 @@ def create_job(
 
         # ...and use directly Job*Config(request.job.config.model_dump_json())
         job_config_args = {
-            "--config": job_config.model_dump_json(exclude_unset=True, exclude_none=True),
```

Comment:
I deleted these configs because this was stripping out parameters that were set to None, which then conflicts with the schema that is used during config creation as well as loading it again inside the inference job. I see these lines were added via

https://github.com/mozilla-ai/lumigator/pull/740/files#diff-59885862ae7b5c8dd1f02df9d48befc70d96c68b03e5308d24a45990f0e5bb22R501

@javiermtorres  would you be able to weigh in here about why this code was added in PR-740?

Reply:
IIRC, including the None values did cause issues to me. If None values are meant as optional and have None values as default in the schema, this should not be an issue. Usually None values can be left out because, in serialization, we don't normally distinguish from a missing JSON key and an explicit `null` for said key; and we can get the list of allowed keys from the schema itself.

Let me know which values cause problems, and we can work it out in the schema. We'll also add a check for those in the backend unit tests.